datasets/lama/README.md

```diff
@@ -0,0 +1,314 @@
+``
+
+The squad config has the following fields:
+
+
+``
+{
+	'id': ...,
+	'sub_label': ...,
+	'obj_label': ...,
+	'negated': ...,
+	'masked_sentence': ...,
+}
+``
+
+The google_re config has the following fields:
+
+
+``
+{
+	'pred': ...,
+	'sub': ...,
+	'obj': ...,
+	'evidences': ...,
+	'judgments': ...,
+	'sub_q': ...,
+	'sub_label': ...
+	'sub_aliases': ...
+	'obj_w': ...
+	'obj_label': ...
+	'obj_aliases': ...
+	'uuid': ...
+	'masked_sentence': ...
+	'template': ...
+	'template_negated': ...
+}
+``
+
+### Data Fields
+
+The trex config has the following fields:
+* uuid: the id
+* obj_uri: a uri for the object slot
+* obj_label: a label for the object slot
+* sub_uri: a uri for the subject slot
+* sub_label: a label for the subject slot
+* predicate_id: the predicate/relationship
+* sub_surface: the surface text for the subject
+* obj_surface: The surface text for the object. This is the word that should be predicted by the [MASK] token.
+* masked_sentence: The masked sentence used to probe, with the object word replaced with [MASK]
+* template: A pattern of text for extracting the relationship, object and subject of the form "[X] some text [Y]", where [X] and [Y] are the subject and object slots respectively. 
+* template_negated: Same as above, except the [Y] is not the object.
+* label: the label for the relationship/predicate
+* description': a description of the relationship/predicate
+* type: a type id for the relationship/predicate
+
+The conceptnet config has the following fields:
+* uuid: the id
+* sub: the subject
+* obj: the object to be predicted.
+* pred: the predicate/relationship
+* obj_label: the object label
+* masked_sentence: The masked sentence used to probe, with the object word replaced with [MASK]
+* negated: same as above, except [MASK] should predict something that is not the object word.
+
+
+The squad config has the following fields:
+* id: the id
+* sub_label: the subject label
+* obj_label: the object label that is being predicted
+* masked_sentence: The masked sentence used to probe, with the object word replaced with [MASK]
+* negated: same as above, except [MASK] should predict something that is not the object word.
+
+
+The google_re config has the following fields:
+
+* uuid: the id
+* pred: the predicate
+* sub: the subject
+* obj: the object
+* evidences: flattened json string that provides evidence for predicate. parse this json string to get more 'snippet' information.
+* judgments: data about judgments
+* sub_q: unknown
+* sub_label: label for the subject
+* sub_aliases: unknown
+* obj_w: unknown
+* obj_label: label for the object
+* obj_aliases: unknown
+* masked_sentence: The masked sentence used to probe, with the object word replaced with [MASK]
+* template: A pattern of text for extracting the relationship, object and subject of the form "[X] some text [Y]", where [X] and [Y] are the subject and object slots respectively. 
+* template_negated: Same as above, except the [Y] is not the object.
+
+

```

Comment:
Can you also mention which fields have missing values that are replace with empty strings ?

Reply:
done